gcloud/datastore/connection.py

```diff
@@ -421,7 +421,7 @@ def allocate_ids(self, dataset_id, key_pbs):
         return list(response.key)
 
     def save_entity(self, dataset_id, key_pb, properties,
```

Comment:
That was on the "Conversation" tab -- looks OK on the "Unified" view of the "Files Changed" tab.